src/ashley/management/commands/sync_group_permissions.py

```diff
@@ -0,0 +1,102 @@
+"""
+This module provides a management command `sync_group_permissions`
+to help ashley operators maintaining a consistent set of forum permission
+across all LTI related groups.
+"""
+
+from django.core.management.base import BaseCommand
+from machina.apps.forum_permission.shortcuts import assign_perm, remove_perm
+from machina.core.db.models import get_model
+
+from ashley.defaults import DEFAULT_FORUM_ROLES_PERMISSIONS
+
+Forum = get_model("forum", "Forum")  # pylint: disable=C0103
+GroupForumPermission = get_model(
+    "forum_permission", "GroupForumPermission"
+)  # pylint: disable=C0103
+LTIContext = get_model("ashley", "LTIContext")  # pylint: disable=C0103
+
+
+class Command(BaseCommand):
+    """
+    Implementation of the sync_group_permission Command.
+    """
+
+    help = (
+        "Update group permissions in database with permissions defined "
+        "in ASHLEY_DEFAULT_FORUM_ROLES_PERMISSIONS. By default, only missing "
+        "permissions are added and no permission is removed."
+    )
+    apply_updates = False
+    remove_extra_permissions = False
+
+    def add_arguments(self, parser):
+        """Set custom arguments for this command."""
```

Comment:
should this command warn about what is going to happen and ask "are you sure" like the `collectstatic` method?

Reply:
I assumed that if the user added the `--apply` argument, he knows what he's doing.